examples/adminapi.py

```diff
@@ -560,6 +562,77 @@ def example_alter_consumer_group_offsets(a, args):
             raise
 
 
+def example_describe_user_scram_credentials(a, args):
+    """
+    Describe User Scram Credentials
+    """
+    futmap = a.describe_user_scram_credentials(args)
+    mechanism_description = {}
+    mechanism_description[ScramMechanism.SCRAM_SHA_256] = "SCRAM-SHA-256"
+    mechanism_description[ScramMechanism.SCRAM_SHA_512] = "SCRAM-SHA-512"
+    mechanism_description[ScramMechanism.UNKNOWN] = "UNKNOWN"
+    for username, fut in futmap.items():
+        print("Username: {}".format(username))
+        try:
+            value = fut.result()
+            for scram_credential_info in value.scram_credential_infos:
+                print("    Mechanism: {} Iterations: {}".
+                      format(scram_credential_info.mechanism,
+                             scram_credential_info.iterations))
+        except KafkaException as e:
+            print("    Error: {}".format(e))
+
+
+def example_alter_user_scram_credentials(a, args):
+    """
+    AlterUserScramCredentials
+    """
+    alterations = []
+    i = 0
+    op_cnt = 0
+    while i < len(args):
+        op = args[i]
+
+        if op == "UPSERT":
+            if i + 5 >= len(args):
```

Comment:
This condition is wrong as salt is optional.

Reply:
salt isn't really optional in this command line example, it needs to be set to empty string to try the None value. Added a comment about that.